chart/values.yaml

```diff
@@ -28,4 +28,4 @@ registry:
   # For token-based logins, use
   #
   # $ echo '{"registrytoken":"9cbaf023786cd7"}' | base64
-  authtoken: changeme
+  authtoken: e30K
```

Comment:
See https://github.com/moby/moby/blob/254fc83cba90ed79c78f4cb0cb33aeeaff492798/client/image_push.go#L54 as an example. Even if the registry isn't backed by auth, an empty auth token still gets shipped and the v2 registry expects that.